templates/web/tfl/report/inspect/extra_details.html

```diff
@@ -0,0 +1,12 @@
+[% IF permissions.report_inspect %]
```

Comment:
If you're having this on its own, it might as well be its own template in the base too then can be shared more easily?